doc/man/po/LINGUAS

```diff
@@ -1,4 +1,4 @@
 de
 fr
 es
-
+it
```

Comment:
Corresponding `doc/man/po/it.po` needs to be added at the same time as this line.